src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -68,9 +68,7 @@
 
 			<Balance token={pageTokenUi} />
 		</div>
-	{/if}
-
-	{#if usdTotal}
+	{:else}
```

Comment:
this is guaranteed to be `'tokens'` route because the type of `HeroRoute` is either `'transactions'` or `'tokens'`